public/app/plugins/datasource/cloudwatch/language/logs/completion/CompletionItemProvider.ts

```diff
@@ -56,6 +71,7 @@ export class LogsCompletionItemProvider extends CompletionItemProvider {
               insertText: `${command} $0`,
               insertTextRules: monaco.languages.CompletionItemInsertTextRule.InsertAsSnippet,
               command: TRIGGER_SUGGEST,
+              kind: monaco.languages.CompletionItemKind.Method,
```

Comment:
Changing these was mostly to make it so that they weren't all using the `Field` color coding. I can change them back if we want.